docs/vagrant/overview.md

```diff
@@ -0,0 +1,7 @@
+# Kubernetes cluster setup with Kubeadm, Terraform and AWS
+
+Terraform, AWS and Kubeadm are used for Virtualization, VMs deployment and K8s Cluster bootstrap:
```

Comment:
This should be swapped with what is inside [docs/aws/overview.md](https://github.com/hlesey/k8s-corabia/pull/49/files#diff-004a6e98d2bdd75c872dd1745e66f66f71bdd635e2046d176e63fd46842f361b) and vice-versa.